integration/config.sh

```diff
@@ -31,7 +31,12 @@ weave_on() {
 scope_end_suite() {
     end_suite
     for host in $HOSTS; do
-        docker_on "$host" rm -f "$(docker_on "$host" ps -a -q)" 2>/dev/null 1>&2 || true
+        docker_on "$host" rm -f "$(docker_on "$host" ps -a -q)" || true
```

Comment:
I thought it was bad to hide errors. It made the issue more difficult to debug.

That doesn't change much in practice because each test in integration/* gets its output discarded in case of success.